resources/quality/ultimaker_s5/um_s5_aa0.8_PLA_Superdraft_Print.inst.cfg

```diff
@@ -17,27 +17,29 @@ cool_fan_speed_max = =100
 cool_min_speed = 2
 gradual_infill_step_height = =3 * layer_height
 infill_line_width = =round(line_width * 0.65 / 0.75, 2)
-infill_pattern = cubic
+infill_pattern = triangles
+layer_height = 0.4
```

Comment:
This is already part of the global quality profile, so it is redundant: https://github.com/Ultimaker/Cura/blob/c8f73d303ea1ec2afb24b9a7808e17254f6da91a/resources/quality/ultimaker_s5/um_s5_global_Superdraft_Quality.inst.cfg#L14